### ISSUE TITLE
Tournament results for stats

### ISSUE THREAD

**Issue by 0x646f6d** [raphi011/scores#1]
It would be nice to be able to consider results of tournaments as well. I was not thinking about submitting every match, more like "i played with xxx on a cup and made the 3rd place" (maybe also use dynamic point algorithm like övv). With this we could make enhanced team statistics like "i made my best results with x".

**Comment by raphi011** [comment]
Good idea, I'd even take it further and import all relevant beach.volleynet data for a user to create statistics :)

**Comment by 0x646f6d** [comment]
We should definately ask for a direct db connection ;) Parsing of that page is horrible!

**Comment by raphi011** [comment]
Can't imagine they would give us a direct con to the db ..
btw it's a wordpress site ... 🤢
